tests/testsuite/vendor.rs

```diff
@@ -1087,6 +1103,7 @@ fn config_instructions_works() {
                 [package]
                 name = "foo"
                 version = "0.1.0"
+                edition = "2021"
```

Comment:
Add a default edition to prevent `[WARNING] no edition set: defaulting to the 2015 edition while the latest is 2021`.